examples/introExample/src/main/java/io/realm/examples/intro/IntroExampleActivity.java

```diff
@@ -41,6 +41,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_realm_basic_example);
+         Realm.init(this);  
```

Comment:
white space is wrong and does not resemble the docs (if I wanted to call this in activity, I'd put it before `super.onCreate()` call)